Project2/style.css

```
@@ -0,0 +1,115 @@
+body {
+    box-sizing: border-box;
+    height: auto;
+    width: 640px;
+}
+
+.mainimage {
+    background-image: url(Images/TheFavouriteWebCardImage.jpg);
+    background-size: cover;
+    height: 380px;
+    position: relative;
+    width: 640px;
+    z-index: 3;
+
+
+}
+
+
+
+.content {
+    background: linear-gradient(to left, #42513F, transparent 190%);
+    height: 100%;
+    width: 640px;
+}
+
+
+.details {
+    left: 410px;
+    padding: 10px;
+    position: relative;
+    top: 1px;
+    width: 200px;
+
+
+}
+
+
+
+
+h1 {
+    bottom: -20px;
+    color: #DB5756;
+    font-size: 20pt;
+    position: relative;
+}
+
+h2 {
+    bottom: 15px;
+    color: #DB5756;
+    font-size: 9pt;
+    position: relative;
+    font-weight: 100;
+    left: 165px;
+    line-height: 8px;
+    word-spacing: 2px;
+}
+
+h3 {
+    bottom: 9px;
+    color: #DB5756;
+    font-size: 11pt;
+    font-weight: 20;
```

What you have now isn't a valid value for `font-weight`; you always want to use 100s (e.g., 100, 200, 300) :
```suggestion
    font-weight: 200;
```